server/hedley/modules/custom/hedley_chw/hedley_chw.module

```diff
@@ -287,7 +287,7 @@ function hedley_chw_participate_in_group($child_id, $adult_id, $group_id) {
  *
  * @throws \EntityMetadataWrapperException
  */
-function hedley_chw_get_village_residents($nid, $range = 1000) {
+function hedley_chw_get_village_residents($nid, $range = 5000) {
```

Comment:
Why do we bump the range? Are we sure it cannot cause memory issues?

Reply:
We bump because we got villages villages with over 1k residents.
From my tests setting limit to 5K does not cause memory issues, as we resolve only the IDs of the patients.